### DISCUSSION TITLE
GlotPress 3.0.0 released

### DISCUSSION THREAD

**Discussion by ocean90** [GlotPress/GlotPress-WP#1399] [Announcements]
### Features
* Various new locales like Ligurian, Tamazight, Nigerian Pidgin, Cornish, Western Balochi., and many more.
* Integrate three new translation warnings for mismatched URLs, missing named placeholders, and unexpected sprintf tokens.
* Reduce reliance on default browser style by updating link and button styles.
* Unify and simplify styling for all tables. 
* Add 'abbreviation' as new part of speech value for glossary entries. 
* Add sort by option for modified date of translation.
* Allow specifying the priority for PO import/exports in the flags.
* Convert translation status filter into list of checkboxes for each status.
* Delete translations if an original gets deleted.

### Bugfixes
* Various fixes for improved plural support and naming fixes.
* Fix removing of starting newline from translations.
* Fix duplicate glossary tooltips for the same term.
* Abort changing translation status when the translation has been altered.
* Fix submitting a translation containing the UTF-8 characters → and ↵.
* Improve calculation of translation status counts. 
* Show new lines and tab characters and wrap non-translatable items for strings with plurals too.
* Use single quotes around string literals in SQL queries.

### Breaking Changes
* GlotPress now requires PHP 7.2 or newer and WordPress 4.6 or newer. 
* iOS .strings file import/exports now are UTF8 encoded instead of UTF16-LE.
* Developers: Various template and design changes.
* Developers: Various unused functions have been removed.
* Developers: Remove all compatibility functions for multibyte string functions. 

For a full list and details of the changes please review the [changelog file](https://github.com/GlotPress/GlotPress/blob/3.0.0/CHANGELOG.md).

## New Contributors
* @deconf made their first contribution in https://github.com/GlotPress/GlotPress/pull/680
* @tobifjellner made their first contribution in https://github.com/GlotPress/GlotPress/pull/761
* @Mte90 made their first contribution in https://github.com/GlotPress/GlotPress/pull/791
* @SVANNER made their first contribution in https://github.com/GlotPress/GlotPress/pull/720
* @jdevalk made their first contribution in https://github.com/GlotPress/GlotPress/pull/619
* @ramonjd made their first contribution in https://github.com/GlotPress/GlotPress/pull/868
* @itssimple made their first contribution in https://github.com/GlotPress/GlotPress/pull/871
* @danieltj27 made their first contribution in https://github.com/GlotPress/GlotPress/pull/874
* @superdav42 made their first contribution in https://github.com/GlotPress/GlotPress/pull/913
* @garretthyder made their first contribution in https://github.com/GlotPress/GlotPress/pull/947
* @maximejobin made their first contribution in https://github.com/GlotPress/GlotPress/pull/1013
* @naokomc made their first contribution in https://github.com/GlotPress/GlotPress/pull/1026
* @akkspros made their first contribution in https://github.com/GlotPress/GlotPress/pull/1023
* @Roshanchy made their first contribution in https://github.com/GlotPress/GlotPress/pull/1029
* @Dedi-Data made their first contribution in https://github.com/GlotPress/GlotPress/pull/1012
* @szepeviktor made their first contribution in https://github.com/GlotPress/GlotPress/pull/1206
* @webaxones made their first contribution in https://github.com/GlotPress/GlotPress/pull/1216
* @gedex made their first contribution in https://github.com/GlotPress/GlotPress/pull/1207
* @paulgoodchild made their first contribution in https://github.com/GlotPress/GlotPress/pull/1221
* @vlad-timotei made their first contribution in https://github.com/GlotPress/GlotPress/pull/1229
* @amieiro made their first contribution in https://github.com/GlotPress/GlotPress/pull/1243
* @xavivars made their first contribution in https://github.com/GlotPress/GlotPress/pull/1305
* @sunxiyuan made their first contribution in https://github.com/GlotPress/GlotPress/pull/1313
* @ckykenken made their first contribution in https://github.com/GlotPress/GlotPress/pull/1309
* @dd32 made their first contribution in https://github.com/GlotPress/GlotPress/pull/1342

**Full Changelog**: https://github.com/GlotPress/GlotPress/compare/2.3.1...3.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/GlotPress/GlotPress/releases/tag/3.0.0'>3.0.0</a>.</em>

**Comment by Mte90** [comment]
After all this years!

**Comment by farhadsakhaei** [comment]
Congratulations ! 😞
